index.html

```diff
@@ -75,6 +75,225 @@ <h1 id="greeting">I'm Conor Glad to See you!</h1>
+
+        <ul class="tonic_list">
+            <li id="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter">
+        </ul>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+                <li class="language_descriptor">html</li>
+                <li class="language_descriptor">css</li>
+                <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        <div class="project_1">
+
+            <div class="project1_img">
+                <img src="img/tonic2.png" alt="tonic">
+            </div>
+
+        <h3 class="project1_title">Tonic</h3>
+
+        <ul class="tonic_list">
+            <li class="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="tonic"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="tonic"> </li>
+            <li> 2015 </li>
+        </ul>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+                <li class="language_descriptor">html</li>
+                <li class="language_descriptor">css</li>
+                <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        <div class="project_1">
+
+            <div class="project1_img">
+                <img src="img/multipoststories2.png" alt="multi-post-stories">
+            </div>
+
+        <h3 class="project1_title">Multi-Post Stories</h3>
+
+        <ul class="tonic_list">
+            <li class="canopy">CANOPY</li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> Back End Dev </li>
+            <li> <img class="counter" src="img/Counter.png" alt="counter"> </li>
+            <li> 2015 </li>
+        </ul>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+        <div class="small_buttons">
+            <ul class="button_list">
+            <li class="language_descriptor">html</li>
+            <li class="language_descriptor">css</li>
+            <li class="language_descriptor">javaScript</li>
+            </ul>
+        </div>
+
+        <button class="see_project" type="button">See Project</button>
+
+
+          
+        </div>
+
+        
+    </section> 
+
+    <section id="contact_info">
+        <div>
+            <h1 id="greeting_2">About Myself</h1>
```

Comment:
- [ ] [OPTIONAL] There should be only one `h1` in an HTML file because of semantics. I kindly suggest that you make only your main title of "I'm Conor Glad to See you!" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] [OPTIONAL] please kindly make changes and replace all other `h1` except your main title

- [ ] [OPTIONAL] kindly make use of headers from `h2` down to `h6` for any other headers apart from your main title of "I'm Conor Glad to See you!" in your HTML file.